travel_platform/smart_ordering_tests.py

```diff
@@ -296,3 +309,14 @@ def test_smart_ordering_filter_rating_desc_limit_zero(self):
                                        order_by="Desc", limit=0)
         expected_result = []
         self.assertEqual(actual_result, expected_result)
+
```

Comment:
As I remember, you've added code with handling of non-existing keys, doesn't it?
But this unit test tests something another.

Please cover added code by test.
Thanks!